.gitignore

```diff
@@ -3,6 +3,7 @@
 WORKSPACE
 # don't check in the build output of the book
 docs/book/book/
+docs/book/src/docs
```

Comment:
Why are we adding it to the gitignore? Why do we need to change the .gitignore to add a new doc section?